react-client/src/Components/DomInfoDialogBox/index.js

```diff
@@ -132,17 +135,18 @@ const DomInfoDialogBox = ({ id, idx, tag, classNames, parent, children, top, lef
           left: `${left}px`,
           border: `3px solid ${borderclr}`,
         }}
-      >
+      >        
         <button id="closedompeeker" className="close-btn-style" onClick={() => onClose(idx, id, uniqueID)}>
           x
         </button>
+       
         <div>
           <div className="dom-header">
             <span className="dom-header-tag">{tag}</span>
-            {id && <span className="dom-header-details">{id}</span>}
-            {domInfo.classNames.filter(obj => obj.name !== ".focused-dom").map((val) => (
+            {id && <span className="dom-header-details">{id}</span>}                       
+              {domUtils.customClassFilter(domInfo.classNames).map((val) => (    
```

Comment:
You can simply add the `.focused-dom` filter inside `extractDomInfo()` function at line 104 and they still give you the same results without introducing another function filter(e.g. `customClassFilter()`) since remember that we use `extractDomInfo()` to prepare the data we need inside DomInfoDialogBox component.

Reply:
because during the bug regarding custom-css the problem occurs after mutation occurs so the program doesnt pass through extractdomInfo and is instead displaying data after state is done after checking for mutation.
Tried to implement extract.DomInfo inside domutils but either it creates a different output or is producing ang error.
since the focus was to filter the class implemented domutils during the map but dom utils returns the entire target so had to create a different function that focuses on just the classname including the  return function

Reply:
Okay let's discuss this on a call later